CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,106 @@
 # Code of Conduct
 
-Facebook has adopted a Code of Conduct that we expect project participants to adhere to. Please [read the full text](https://code.fb.com/codeofconduct/) so that you can understand what actions will and will not be tolerated.
+This code of conduct outlines our expectations for participants within
+the osquery community, as well as steps to reporting unacceptable
+behavior. We are committed to providing a welcoming and inspiring
+community for all and expect our code of conduct to be honored. Anyone
+who violates this code of conduct may be banned from the community.
+
+Our community strives to:
+
+* **Be friendly and patient.**
+* **Be welcoming**: We strive to be a community that welcomes and
+  supports people of all backgrounds and identities. This includes,
+  but is not limited to members of any race, ethnicity, culture,
+  national origin, color, immigration status, social and economic
+  class, educational level, sex, sexual orientation, gender identity
+  and expression, age, size, family status, political belief,
+  religion, and mental and physical ability.
+* **Be considerate**: Your work will be used by other people, and you
+in turn will depend on the work of others. Any decision you take will
+affect users and colleagues, and you should take those consequences
+into account when making decisions. Remember that we’re a world-wide
+community, so you might not be communicating in someone else’s primary
+language.  Be respectful: Not all of us will agree all the time, but
```

Comment:
I think that the `Be respectful` was on its own line in order to show as a list item